python/examples/bindings/se2-bindings/wasm-wrapper-c/build-wasm.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+if [[ ! -v WASI_SDK_PATH ]]; then
+    echo "WASI_SDK_PATH is required. Download wasi-sdk and set the variable"
+    exit 1
+fi
+
+export TARGET_DIR=target/wasm32-wasi
+if [[ "$1" == "--clean" ]]; then
+    rm -rf ${TARGET_DIR}
+fi
+
+mkdir -p ${TARGET_DIR}/deps 2>/dev/null
+if [ -f ${TARGET_DIR}/deps/include/python3.11/Python.h -a -f ${TARGET_DIR}/deps/lib/wasm32-wasi/libpython3.11.a ]; then
+    echo "Dependencies already downloaded. Reusing..."
+else
+    curl -sL https://github.com/assambar/webassembly-language-runtimes/releases/download/python%2F3.11.1%2B20230223-8a6223c/libpython-3.11.1.tar.gz | tar xzv -C ${TARGET_DIR}/deps
```

Comment:
I know this is an experiment, but it seems to me we will need to access resources from time to time. It would be great to have a way to link to the latest version easily. Let's discuss how we can provide such experience :)

Reply:
Agreed. We will have to have some redirect or something. I will update this PR after I publish Python3 so no rogue links are there.